tests/elastio_snap.py

```diff
@@ -42,14 +51,25 @@
 
 lib = ffi.dlopen("../lib/libelastio-snap.so")
 
-
-def setup(minor, device, cow_file, fallocated_space=0, cache_size=0):
+if sys.version_info >= (3, 6):
+    class State(IntFlag):
+        SNAPSHOT = 1
+        ACTIVE = 2
+        UNVERIFIED = 4
+else:
+    class State:
+        SNAPSHOT = 1
+        ACTIVE = 2
+        UNVERIFIED = 4
```

Comment:
May be use this class instead of IntFlag for  python >= 3.6? For decrease changes

Reply:
Good idea. Removed `class State(IntFlag)` and added a comment about Python 3.6.